common/config/defaults.go

```diff
@@ -12,25 +12,30 @@ const (
 	DefaultLockExpiry              = time.Hour * 1
 	DefaultPort                    = 9090
 	DefaultEnvironmentInstanceType = "t2.small"
-	DefaultEnvironmentType         = models.EnvironmentTypeDynamic
 	DefaultEnvironmentOS           = "linux"
 	DefaultServiceScale            = 1
+	DefaultLoadBalancerType        = models.ApplicationLoadBalancerType
+	DefaultTargetGroupPort         = 80
+	DefaultTargetGroupProtocol     = "HTTP"
+
+	// https://docs.aws.amazon.com/AmazonECS/latest/developerguide/platform_versions.html
+	DefaultFargatePlatformVersion = "1.0.0"
```

Comment:
Should we be targeting the 1.1.0? It looks like it has nice things that we can make use of. 

```
Fargate Platform Version‐1.1.0
Added support for Amazon ECS Task Metadata Endpoint.
Added support for Health Check.
Added support for Service Discovery
```

I would make a case for using the Latest but this sentence from AWS docs doesn't make any sense
> If you specify LATEST, your tasks use the most current platform version available, which may not be the most recent platform version.

What is the most 'current' vs the 'recent'?

Reply:
That _is_ confusing. I bet we could update to use `1.1.0`; I had originally been using `LATEST`, but @diemonster cautioned against it because of how volatile the Fargate launch type will likely be.